### ISSUE TITLE
Install fails on python3.8

### ISSUE THREAD

**Issue by adrienball** [scrapinghub/python-crfsuite#116]
### Description
I get an error when trying to install `python-crfsuite` with python3.8:

```
$ python --version                                                                                                                                   
Python 3.8.1
$ pip install python-crfsuite                                                                                                                        adrien@adrien
Looking in indexes: https://pypi.python.org/simple/, https://pypi:****@nexus-repository.snips.ai/repository/pypi-internal/simple
Collecting python-crfsuite
  Using cached https://files.pythonhosted.org/packages/73/b5/d9640656386b28ca181410f292ffcc3f1c39d9438f0796f874fd5d65ee34/python-crfsuite-0.9.6.tar.gz
Building wheels for collected packages: python-crfsuite
  Building wheel for python-crfsuite (setup.py) ... error
  ERROR: Command errored out with exit status 1:
   command: /Users/adrien/dev/test/venv38/bin/python3.8 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/private/var/folders/36/y4kq1zd54d13g7gnv58nt4lw0000gn/T/pip-install-msacb856/python-crfsuite/setup.py'"'"'; __file__='"'"'/private/var/folders/36/y4kq1zd54d13g7gnv58nt4lw0000gn/T/pip-install-msacb856/python-crfsuite/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /private/var/folders/36/y4kq1zd54d13g7gnv58nt4lw0000gn/T/pip-wheel-k72gxbja --python-tag cp38
       cwd: /private/var/folders/36/y4kq1zd54d13g7gnv58nt4lw0000gn/T/pip-install-msacb856/python-crfsuite/
  Complete output (25 lines):
  running bdist_wheel
  running build
  running build_py
  creating build
  creating build/lib.macosx-10.9-x86_64-3.8
  creating build/lib.macosx-10.9-x86_64-3.8/pycrfsuite
  copying pycrfsuite/_dumpparser.py -> build/lib.macosx-10.9-x86_64-3.8/pycrfsuite
  copying pycrfsuite/_logparser.py -> build/lib.macosx-10.9-x86_64-3.8/pycrfsuite
  copying pycrfsuite/__init__.py -> build/lib.macosx-10.9-x86_64-3.8/pycrfsuite
  running build_ext
  building 'pycrfsuite._pycrfsuite' extension
  creating build/temp.macosx-10.9-x86_64-3.8
  creating build/temp.macosx-10.9-x86_64-3.8/pycrfsuite
  creating build/temp.macosx-10.9-x86_64-3.8/crfsuite
  creating build/temp.macosx-10.9-x86_64-3.8/crfsuite/lib
  creating build/temp.macosx-10.9-x86_64-3.8/crfsuite/lib/crf
  creating build/temp.macosx-10.9-x86_64-3.8/crfsuite/lib/crf/src
  creating build/temp.macosx-10.9-x86_64-3.8/crfsuite/swig
  creating build/temp.macosx-10.9-x86_64-3.8/crfsuite/lib/cqdb
  creating build/temp.macosx-10.9-x86_64-3.8/crfsuite/lib/cqdb/src
  creating build/temp.macosx-10.9-x86_64-3.8/liblbfgs
  creating build/temp.macosx-10.9-x86_64-3.8/liblbfgs/lib
  gcc -Wno-unused-result -Wsign-compare -Wunreachable-code -fno-common -dynamic -DNDEBUG -g -fwrapv -O3 -Wall -arch x86_64 -g -Icrfsuite/include/ -Icrfsuite/lib/cqdb/include -Iliblbfgs/include -Ipycrfsuite -I/Library/Frameworks/Python.framework/Versions/3.8/include/python3.8 -c pycrfsuite/_pycrfsuite.cpp -o build/temp.macosx-10.9-x86_64-3.8/pycrfsuite/_pycrfsuite.o -std=c99
  error: invalid argument '-std=c99' not allowed with 'C++'
  error: command 'gcc' failed with exit status 1
  ----------------------------------------
  ERROR: Failed building wheel for python-crfsuite
  Running setup.py clean for python-crfsuite
Failed to build python-crfsuite
Installing collected packages: python-crfsuite
    Running setup.py install for python-crfsuite ... error
    ERROR: Command errored out with exit status 1:
     command: /Users/adrien/dev/test/venv38/bin/python3.8 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/private/var/folders/36/y4kq1zd54d13g7gnv58nt4lw0000gn/T/pip-install-msacb856/python-crfsuite/setup.py'"'"'; __file__='"'"'/private/var/folders/36/y4kq1zd54d13g7gnv58nt4lw0000gn/T/pip-install-msacb856/python-crfsuite/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /private/var/folders/36/y4kq1zd54d13g7gnv58nt4lw0000gn/T/pip-record-hw20tx6k/install-record.txt --single-version-externally-managed --compile --install-headers /Users/adrien/dev/test/venv38/bin/../include/site/python3.8/python-crfsuite
         cwd: /private/var/folders/36/y4kq1zd54d13g7gnv58nt4lw0000gn/T/pip-install-msacb856/python-crfsuite/
    Complete output (25 lines):
    running install
    running build
    running build_py
    creating build
    creating build/lib.macosx-10.9-x86_64-3.8
    creating build/lib.macosx-10.9-x86_64-3.8/pycrfsuite
    copying pycrfsuite/_dumpparser.py -> build/lib.macosx-10.9-x86_64-3.8/pycrfsuite
    copying pycrfsuite/_logparser.py -> build/lib.macosx-10.9-x86_64-3.8/pycrfsuite
    copying pycrfsuite/__init__.py -> build/lib.macosx-10.9-x86_64-3.8/pycrfsuite
    running build_ext
    building 'pycrfsuite._pycrfsuite' extension
    creating build/temp.macosx-10.9-x86_64-3.8
    creating build/temp.macosx-10.9-x86_64-3.8/pycrfsuite
    creating build/temp.macosx-10.9-x86_64-3.8/crfsuite
    creating build/temp.macosx-10.9-x86_64-3.8/crfsuite/lib
    creating build/temp.macosx-10.9-x86_64-3.8/crfsuite/lib/crf
    creating build/temp.macosx-10.9-x86_64-3.8/crfsuite/lib/crf/src
    creating build/temp.macosx-10.9-x86_64-3.8/crfsuite/swig
    creating build/temp.macosx-10.9-x86_64-3.8/crfsuite/lib/cqdb
    creating build/temp.macosx-10.9-x86_64-3.8/crfsuite/lib/cqdb/src
    creating build/temp.macosx-10.9-x86_64-3.8/liblbfgs
    creating build/temp.macosx-10.9-x86_64-3.8/liblbfgs/lib
    gcc -Wno-unused-result -Wsign-compare -Wunreachable-code -fno-common -dynamic -DNDEBUG -g -fwrapv -O3 -Wall -arch x86_64 -g -Icrfsuite/include/ -Icrfsuite/lib/cqdb/include -Iliblbfgs/include -Ipycrfsuite -I/Library/Frameworks/Python.framework/Versions/3.8/include/python3.8 -c pycrfsuite/_pycrfsuite.cpp -o build/temp.macosx-10.9-x86_64-3.8/pycrfsuite/_pycrfsuite.o -std=c99
    error: invalid argument '-std=c99' not allowed with 'C++'
    error: command 'gcc' failed with exit status 1
    ----------------------------------------
ERROR: Command errored out with exit status 1: /Users/adrien/dev/test/venv38/bin/python3.8 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/private/var/folders/36/y4kq1zd54d13g7gnv58nt4lw0000gn/T/pip-install-msacb856/python-crfsuite/setup.py'"'"'; __file__='"'"'/private/var/folders/36/y4kq1zd54d13g7gnv58nt4lw0000gn/T/pip-install-msacb856/python-crfsuite/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /private/var/folders/36/y4kq1zd54d13g7gnv58nt4lw0000gn/T/pip-record-hw20tx6k/install-record.txt --single-version-externally-managed --compile --install-headers /Users/adrien/dev/test/venv38/bin/../include/site/python3.8/python-crfsuite Check the logs for full command output.
```

## Environment
MacOS 10.14.6
Python 3.8.1

**Comment by eightypercentcoffee** [comment]
same problem on here
MacOS 10.15.3
Python 3.8.2

**Comment by kmike** [comment]
Could you please check it? It should be fixed in 0.9.7 release.

**Comment by adrienball** [comment]
I confirm that `0.9.7` fixed the issue. 
Thanks
